ui/chat/src/discussion.ts

```diff
@@ -82,6 +85,19 @@ function renderInput(ctrl: ChatCtrl): VNode | undefined {
   });
 }
 
+function prependChatInput(prefix: string): void {
+  const chatInput = document.querySelector('input.mchat__say') as HTMLInputElement;
+  if (chatInput.value.startsWith(prefix)) return;
+  const newVal = prefix + chatInput.value;
+  chatInput.focus();
+  chatInput.select();
+  if (!document.execCommand('insertText', false, newVal)) {
```

Comment:
`document.execCommand` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)

Reply:
There are a few alternatives but none are fully ideal (since there's no direct replacement yet for `execCommand`):
1) Just use the code I have inside that if block. The only problem is if the user then does ctrl+z, the '@username' won't be removed.
2) Use a third party library (instead of `execCommand`) to mimic user input so that it's included in the browser's ctrl+z stack.